### PR TITLE
render video and audio where appropriate

### PR DESCRIPTION
**Pull request by gestchild**
Building on #8162

This renders the appropriate content for a guide based on the type in the url, e.g.
Screenshot 2022-08-02 at 18 29 52

Screenshot 2022-08-02 at 18 30 14

Screenshot 2022-08-02 at 13 00 56

**There are a few todos and so some tidying up needed, but thought it would be good to get in and we could maybe split the remaining work.**

### CODE REVIEW

File: content/webapp/services/prismic/transformers/exhibition-guides.ts [hunk 1]
```diff
@@ -108,9 +128,11 @@ export function transformExhibitionGuide(
         caption: (component.caption && asRichText(component.caption)) || [],
         transcription:
           (component.transcript && asRichText(component.transcript)) || [],
-        audioWithDescription: component['audio-with-description'],
-        audioWithoutDescription: component['audio-without-description'],
-        bsl: component['bsl-video'],
+        audioWithDescription: component['audio-with-description'], // TODO make the same as other audio transforms
```
Comment (melanierogan) [inline content/webapp/services/prismic/transformers/exhibition-guides.ts:131]: good shout

File: content/webapp/pages/exhibition-guide.tsx [hunk 2]
```diff
@@ -86,15 +88,67 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
...
+          <li key={index}>
+            <h2>
+              {number}. {title}
+            </h2>
+            {hasContentOfDesiredType ? (
+              <>
+                {type === 'audio-with-descriptions' &&
+                  audioWithDescription.url && (
+                    <AudioPlayer
+                      title={stop.title} // TODO option not to display title
```
Comment (melanierogan) [inline content/webapp/pages/exhibition-guide.tsx:117]: Do you mean giving editors an option in Prismic UI, what did you have in mind?

**Review by melanierogan — APPROVED** [review]
👍

File: content/webapp/pages/exhibition-guide.tsx (hunk 2 above)
Comment (gestchild) [reply content/webapp/pages/exhibition-guide.tsx:117]: No, It's just that sometimes we'll want the component to render the title and other times not, but we'll know that at the point we use it. If you look at the screenshot above, you can see the title gets displayed twice, once by the code for the 'Stop' and once by the AudioPlayer component itself. In this instance, I think we'll not want the AudioPlayer to display it.

**Pull request merged**